core/runtime/pom.xml

```diff
@@ -129,6 +129,10 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-builder</artifactId>
```

Comment:
Is it safe to add this dependency?

@gsmet wdyt?

Reply:
No, it's not. It comes with a lot of classes including all our build chain classes.

I think we'd better include a constant in what is generated here: `MainClassBuildStep` given we already have things using it there.

Reply:
@cescoffier very good catch btw.

Reply:
Ah man, sorry I missed this one

Reply:
> I think we'd better include a constant in what is generated here: `MainClassBuildStep` given we already have things using it there.

Done. Thanks for the pointer @gsmet.